src/protocol.rs

```diff
@@ -34,9 +34,8 @@ const OFFSET_FETCH_KEY: i16 = 9;
 const VERSION: i16 = 0;
 
 const FETCH_MAX_WAIT_TIME: i32 = 100;
-const FETCH_MIN_BYTES: i32 = 4096;
-const FETCH_BUFFER_SIZE_BYTES: i32 = 4096;
-const MAX_FETCH_BUFFER_SIZE_BYTES: i32 = FETCH_BUFFER_SIZE_BYTES * 8;
+const FETCH_MIN_BYTES: i32 = 1;
```

Comment:
Yeah, sorry - committed to the wrong branch - will revert all the changes
other than the error handling
On Aug 25, 2015 9:22 PM, "Frank Denis" <notifications@github.com> wrote:

> In src/protocol.rs
> <https://github.com/spicavigo/kafka-rust/pull/17#discussion_r37945323>:
>
> > @@ -34,9 +34,8 @@ const OFFSET_FETCH_KEY: i16 = 9;
> >  const VERSION: i16 = 0;
> >
> >  const FETCH_MAX_WAIT_TIME: i32 = 100;
> > -const FETCH_MIN_BYTES: i32 = 4096;
> > -const FETCH_BUFFER_SIZE_BYTES: i32 = 4096;
> > -const MAX_FETCH_BUFFER_SIZE_BYTES: i32 = FETCH_BUFFER_SIZE_BYTES * 8;
> > +const FETCH_MIN_BYTES: i32 = 1;
>
> I think it was a temporary test that eventually stuck around in the pull
> request :) frew@c5a0b5e
> <https://github.com/frew/kafka-rust/commit/c5a0b5eff26de53c812b949c310af06b41d785d7>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spicavigo/kafka-rust/pull/17/files#r37945323>.
>